.github/workflows/python-build.yml

```
@@ -28,6 +28,8 @@ jobs:
         toxenv: ["docs", "readme"]
     runs-on: ubuntu-latest
     steps:
+      - name: Install dot
+        run: sudo apt install -y graphviz
```

The name does not seem to be aligned with the command.  It is self-documenting to just remove the name.
```suggestion
      - run: sudo apt install -y graphviz
```